broker/consumer/vm_startup.sh

```diff
@@ -87,17 +87,28 @@ do
     # which kills the while loop. no working suggestions found.
     # passing the error with `|| true`
 
+    
+    IFS=', ' read -r -a KAFKA_TOPIC_ARRAY <<< "$KAFKA_TOPIC"
+    KAFKA_TOPIC_PRESENT=()
```

Comment:
It is important that this variable  be defined right before the next for `loop` starts. Make this more clear to future readers by moving this down between the current lines 94 and 95 so that it's actually part of the same block of code. 

Additionally, using all caps to name a variable is typically only done for environment variables and module level variables. This is neither, so use snake case.